en/components/localization.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Localization - Native Angular | Ignite UI for Angular
+_description: The Ignite UI for Angular Localization mechanism provides the ability to change/localize strings in the components.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Native Angular Components
+---
+
+## Localization
+
+With only a few lines of code, users can easily localize the strings in Ignite UI for Angular components.
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="localization-sample-1-iframe" src='{environment:demosBaseUrl}/localization-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="localization-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Usage
+
+#### Localize entire application
+
+To get all available resource strings, there is a global function `getCurrentResourceStrings`, which returns an `IResourceStrings` object.
```

Comment:
We will have to write somewhere that `IResourceStrings` will live in other repo and should be imported from there.